JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt

```diff
@@ -65,82 +71,75 @@ import com.example.jetnews.model.Paragraph
 import com.example.jetnews.model.ParagraphType
 import com.example.jetnews.model.Post
 import com.example.jetnews.ui.ThemedPreview
-import com.example.jetnews.ui.VectorImage
 import com.example.jetnews.ui.darkThemeColors
 
 private val defaultSpacerSize = 16.dp
 
 @Composable
-fun PostContent(modifier: Modifier = Modifier.None, post: Post) {
-    val typography = MaterialTheme.typography()
+fun PostContent(post: Post, modifier: Modifier = Modifier) {
+    val typography = MaterialTheme.typography
```

Comment:
Inline this? Only seems to be used once.

Reply:
Done.